PowerEditor/src/Notepad_plus.h

```diff
@@ -360,8 +360,8 @@ friend class FileManager;
 	//Synchronized Scolling
 	struct SyncInfo final
 	{
-		int _line = 0;
-		int _column = 0;
+		INT_PTR _line = 0;

```

Comment:
Consider using fixed-size integers from <cstdint> header.
https://en.cppreference.com/w/cpp/types/integer
For example: `int64_t`, `uint64_t`. Or use some typedef like `using line_t = int64_t`
This will make it more obvious and it will be less dependent on Win32 headers.

Reply:
@mere-human 
I don't see your point.

Reply:
I suggested using fixed size integer types from standard C++ headers instead of `INT_PTR` from Windows header.
This could add more clarity and reduce ambiguity.
Also there's `intptr_t` .
What was the idea behind using `INT_PTR`?

Reply:
There's no idea behind INT_PTR, just the type I found which is signed "size_t" but indeed, intptr_t looks better.

Reply:
By the way, `intptr_t` is 4 bytes in x32 app and 8 bytes in x64 app. Is it expected?
What is the expected behavior in x32 version of the app for the fixed bug?

Reply:
Yes, it's expected.
In this PR, the file length can be beyond 4bytes for x64 binary so the position of cursor can be beyond 4bytes, therefore we need 8 bytes to hold this value. 32bits binary keeps the same behaviour and nothing to be fixed.